### ISSUE TITLE
Alternative File Upload Method via Volume Mount

### ISSUE THREAD

**Issue by rajavishah** [bilayer-containers/bilayers#31]
As discussed with @gnodar01, we are introducing an additional option for file uploads using volume mounts.

Previously, the process for uploading files involved creating a temporary directory on the fly and copying all user-provided files into this newly created folder. This approach can be unnecessarily resource-intensive due to the copy operation. 

To optimize this, we are now providing an alternative option to use a volume mount with their Docker image.

#### New Input Flags:
- *volume_mount*: True
- *folder_name*: “/bilayers/input_images”

If a file input should not allow the volume mount option, the user can specify:
- *volume_mount*: False

In this case, the application will automatically revert to the temporary directory approach.

### Considerations:
The main drawback of this approach is that users must know in advance which inputs need to be volume mounted and what the corresponding folder names are. Additionally, when starting the Docker container, users need to specify the correct host path via Docker Desktop.

**Comment by rajavishah** [comment]
Following an in-person discussion, we decided to simplify the user experience for those using Gradio by removing the need for volume mounting. Instead, users will directly see the output displayed. For more advanced users who prefer Jupyter notebooks, we are only offering the volume-mount option. These configurations can be updated in the future if needed.

As a result, we have removed the `volume_mount` flag from the `config.yaml` file. The logic for handling this has been written and committed to GitHub. Future developers can refer to the commit history for the implementation details if needed.
